### PR TITLE
Bubble sort

### PR DESCRIPTION
**Pull request by christianotieno**

### CODE REVIEW

File: bubble_sort.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+def bubble_sort(arr)
```
Comment (Ispirett) [inline bubble_sort.rb]: I get this error when testing you code.
```
LocalJumpError:
       no block given (yield)

``
`remove yield(arr) and replace with yield`

**Review by Ispirett — CHANGES_REQUESTED** [review]
## status changes required :1st_place_medal: 
great work @ChristianOtieno  @mikenath223  so far you are doing great. :+1: 

There are just a few adjustments to be made.

**Required**
- [x] You are missing the bubble_sort_by method. Please implement this

File: bubble_sort.rb (hunk 1 above)
Comment (mikenath223) [reply bubble_sort.rb]: Changes implemented.
Thanks.

**Review by BalintMendli — APPROVED** [review]
Hi @ChristianOtieno @mikenath223 ,

Great job. 👏 

It is a 
### 👍 

You can go ahead and merge the project.

**Comment by mikenath223** [comment]
Alright.
Thanks

**Pull request merged**